src/lib.rs

```diff
@@ -31,6 +68,25 @@ pub fn get() -> usize {
 ///
 /// If not possible on the particular architecture, returns same as `get()`
 /// which is the logical CPUs.
+/// 
+/// # Examples
+/// 
+/// ```
+/// let logical_cpus = num_cpus::get();
+/// let physical_cpus = num_cpus::get();
```

Comment:
I suspect this should be `num_cpus::get_physical()`?

Reply:
yup :)